styles.css

```diff
@@ -0,0 +1,872 @@
+}
+
+.services__block--image {
+    width: 10em;
+    margin: 1em 1em 0.5em 0;
+}
+
+/* --------------- 4.7 Footer-Form------------------- */
+
+#footer__outer-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-image: var(--main-gradient-color);
+    width: 100%;
+}
+
+.footer__inner-container {
+    width: 100%;
+    flex-basis: 90vh;
+}
+
+.footer__form,
+.footer__inner-container,
+.footer__form--form-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.footer__form {
+    width: 95%;
+    margin-top: 2em;
+    margin-bottom: 2em;
+}
+
+.footer__form--form-section {
+    width: 100%;
+    margin: 2em 0 2em 0;
+}
+
+.form__input {
+    width: 100%;
+}
+
+label {
+    margin: 1em;
+}
+
+input[type=text],
+textarea {
+    padding: 1em;
+    border-radius: 2px;
+    border: none;
+    resize: none;
+}
+
+/* --------------- 4.8 Footer-Copyright-section------------------- */
+
+.footer__copyright {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    background-color: var(--main-font-color-dark);
+    height: 8em;
+}
+
+/* ------------------------------- 5. Bonus add-on features --------------------------------------- */
+
+/* --------------- 5.1 Pseudo elements-Text-items ------------------- */
+
+.main__about-section{
+    position: relative;
+}
+
+.main__paragraph--about-section::before{
+    content: "hello";
+    display: block;
+    position: absolute;
+    top: -40px;
+    width: 90%;
+    height: 30%;
+    text-align: center;
+    font-size: 25em;
+    font-weight: 700;
+    color: var(--main-background-color-darker);
+    z-index: -1;
+}
+
+.main__paragraph--about-section {
+    z-index: 2;
+}
+
+#main__content-section {
+    position: relative;
+}
+
+.main__content-article--pseudo::after {
+    content: "work";
+    display: block;
+    position: absolute;
+    top: 10px;
+    width: 90%;
+    height: 30%;
+    text-align: center;
+    font-size: 25em;
+    font-weight: 700;
+    color: var(--main-font-color-light);
+    z-index: -1;
+}
+
+.main__content-article {
+    z-index: 2;
+}
+
+#main__services--inner-container {
+    position: relative;
+}
+
+#main__services--inner-container::before{
+    content: "services";
+    display: block;
+    position: absolute;
+    top: -150px;
+    width: 90%;
+    height: 30%;
+    text-align: center;
+    font-size: 15em;
+    font-weight: 700;
+    color: var(--main-background-color-darker);
+    z-index: -1;
+}
+
+/* --------------- 5.2 Pseudo elements-Sqewed-triangles ------------------- */
+
+.header--pseudo-skew::after,
+.main__content-section--pseudo-skew::before,
+.services__inner-container--pseudo-skew::before,
+.footer-section--pseudo-skew::before {
+    content: "";
+    display: block;
+    width: 0;
+    height: 0;
+    border-top: 2.5em solid transparent;
```

Comment:
In principe is een pseudo element hier maken een beetje overbodig! Je skewer kon namelijk enkel opgemaakt worden uit de borders! En in dit geval wil je niet dat de skewer een position van relatief heeft, want relatief valt eigenlijk voor het parent element. We willen het op position absoluut hebben omdat we dan het child element (de skewer) kunnen plaatsen waar we willen! :)